client/components/domains/register-domain-step/index.jsx

```diff
@@ -1439,9 +1439,11 @@ class RegisterDomainStep extends Component {
 	};
 
 	showAvailabilityErrorMessage( domain, error, errorData ) {
-		const { DOTBLOG_SUBDOMAIN, TRANSFERRABLE } = domainAvailability;
+		const { DOTBLOG_SUBDOMAIN, TRANSFERRABLE, RECENT_REGISTRATION_LOCK_NOT_TRANSFERRABLE } =
+			domainAvailability;
 		if (
 			( TRANSFERRABLE === error && this.state.lastDomainIsTransferrable ) ||
+			RECENT_REGISTRATION_LOCK_NOT_TRANSFERRABLE === error ||
```

Comment:
This additional condition is to not show an error notice if a recently registered domain is searched. In these cases, we should only show the "<domain> is already registered. Do you own it?" notice.